packages/mdc-chips/_mixins.scss

```diff
@@ -438,6 +465,15 @@ $mdc-chip-ripple-target: ".mdc-chip__ripple";
     padding-right: $padding;
     padding-left: $padding;
   }
+
+  $margin: 0 - $padding;
```

Comment:
nit: `$margin: -$padding;`

Reply:
That whole chunk should have been removed